src/containers/MyNdla/Folders/FoldersPage.tsx

```diff
@@ -113,6 +120,11 @@ const FoldersPage = () => {
     () => (selectedFolder ? selectedFolder.subfolders : (data?.folders.folders as GQLFolder[]) ?? []),
     [selectedFolder, data?.folders],
   );
+  const sharedByOthersFolders: GQLFolder[] = useMemo(
+    () => (selectedFolder ? selectedFolder.subfolders : (data?.folders.sharedFolders as unknown as GQLFolder[]) ?? []),
+    [selectedFolder, data?.folders.sharedFolders],
+  );
```

Comment:
Dette er litt rart. For om selectedFolder har en verdi, så er du ikkje på rotnivået og dermed kan du ikkje ha mapper andre har delt synlig. Og det gir ikkje meining å putte subfolders inn her heller så bedre å returnere [] då.